sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -8,7 +8,7 @@
 
 import type { AbortSignalLike } from '@azure/abort-controller';
 import type { AccessToken } from '@azure/core-auth';
-import type { AzureLogger } from '@azure/logger';
+import { AzureLogger } from '@azure/logger';
```

Comment:
Not a huge deal, but I would not expect any public API changes - curious where this came from?

Reply:
I would also recommend someone more familiar with core-rest-pipeline to review this package's changes closely - it seems ok, but what do I know I just work here :D